components/MailChimpForm.js

```diff
@@ -75,6 +76,12 @@ const CustomForm = ({ status, message, onValidated }) => {
   );
 };
 
+CustomForm.propTypes = {
+  status: PropTypes.string,
+  message: PropTypes.string,
+  onValidated: PropTypes.func,
```

Comment:
Should we always expect an `onValidated`?

```suggestion
  onValidated: PropTypes.func.isRequired,
```